solr/contrib/ltr/src/java/org/apache/solr/ltr/FeatureLogger.java

```diff
@@ -192,6 +192,33 @@ public String makeFeatureVector(LTRScoringQuery.FeatureInfo[] featuresInfo) {
       return features;
     }
 
+    /*
```

Comment:
Good question. `TestFeatureLogging` is not the only test using these methods, both now and also going forward.